keys/keys_test.go

```diff
@@ -0,0 +1,39 @@
+package keys
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	TestMnenomic     = "equip town gesture square tomorrow volume nephew minute witness beef rich gadget actress egg sing secret pole winter alarm law today check violin uncover"
+	TestExpectedAddr = "kava1gflyk57p2ppflqjszyak29x62wuw955eqrzcmy"
```

Comment:
When i manually test that mneomic using `kvcli keys add dry --dry-run --recover` I get kava15qdefkmwswysgg4qxgqpqr35k3m49pkx2jdfnw

Reply:
see below, there's a typo in the BIP44Prefix

Reply:
https://github.com/Kava-Labs/go-sdk/pull/1/commits/64d1867e55f9a57b0d3c1c7b21d937ee0b874d62